src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28798.cs

```diff
@@ -0,0 +1,22 @@
+#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+internal class Issue28798 : _IssuesUITest
+{
+	public Issue28798(TestDevice device) : base(device) { }
+
+	public override string Issue => "Controls Disappear When WebView is Used with Hardware Acceleration Disabled in Android";
+
+	[Test]
+	[Category(UITestCategories.WebView)]
+	public void ControlsShouldRemainVisibleWithWebViewWhenHardwareAccelerationIsDisabled()
```

Comment:
This test is failing on Android.
`Snapshot different than baseline: ControlsShouldRemainVisibleWithWebViewWhenHardwareAccelerationIsDisabled.png (21.12% difference)`
![image](https://github.com/user-attachments/assets/c86f1455-6edc-4615-8009-1d208127ca92)

Could you review it?